src/services/healthcareProfessionalService.ts

```diff
@@ -75,10 +75,13 @@ Promise<Result<gqlTypes.HealthcareProfessional[]>> {
         const hasSpecialtiesFilter = filters.specialties && filters.specialties.length > 0
         const hasSpokenLanguagesFilter = filters.spokenLanguages && filters.spokenLanguages.length > 0
         const hasDegreesFilter = filters.degrees && filters.degrees.length > 0
+        const hasNamesFilter = filters.names && filters.names.length > 0
```

Comment:
can remove the `> 0` part

Reply:
Is the length not necessary? Since the filters.names is an array with an object inside? So when I send an empty array it would pass the check but without containing the name?

Reply:
I meant `filters.names.length > 0` runs the same as `filters.names.length` in a boolean check

Reply:
Of course since 0 is falsy 🙇‍♀️